queries.sql

```diff
@@ -0,0 +1,54 @@
+-- queries for the given questions
+--Question 1:
+SELECT sender_id, receiver_id, amount, created_at 
+FROM Transactions
+WHERE created_at <= now() AND created_at >= given_date;
+
+SELECT account_id, amount, balance, created_at 
+FROM Withdraws/Deposits
+WHERE created_at <= now() AND created_at >= given_date;
+
+--Question 2
+--- for converting into json format
+SELECT sender_id, receiver_id, amount, created_at 
+FROM Transactions
+WHERE created_at <= now() AND created_at >= given_date
+FOR JSON AUTO;
+
+SELECT account_id, amount, balance, created_at 
+FROM Withdraws/Deposits
+WHERE created_at <= now() AND created_at >= given_date
+FOR JSON AUTO;
+
+--Question 3
+--loan account report
+SELECT customer_id, type, amount, interest, period, balance, monthly_emi, monthly_emi_remaining, monthly_emi_date
+FROM Loans
+WHERE customer_id = given_id;
+
+--Question 4
+--a
+SELECT id, account_id, SUM(amount) AS amount
```

Comment:
All 4 questions should be answered in one sql query

Reply:
Okay, I will try to write it in a single query.